entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java

```diff
@@ -351,6 +351,60 @@ public ResponseEntity<EntityIdResponse> triggerMetricsUpdateMultipleEntities(
     return scheduleBatchUpdates(request, entityIds, ScheduledUpdateType.METRICS_UPDATE);
   }
 
+  /**
+   * Synchronize Organizations from Zoho
+   *
+   * @param type type of entity
+   * @param identifier entity id
+   * @param action used with either “enable” or “disable” to mark an entity to be used or not for
+   *     enrichment using labels.
+   * @param request
+   * @return
+   * @throws HttpException
+   */
+  @ApiOperation(
+      value = "Enable or disable the use of the entity for enrichment",
+      nickname = "enableDisableForEnrich",
+      response = java.lang.Void.class)
+  @PostMapping(
+      value = "/{type}/{identifier}/management/enrich",
+      produces = {MediaType.APPLICATION_JSON_VALUE, HttpHeaders.CONTENT_TYPE_JSONLD})
+  public ResponseEntity<String> enableDisableForEnrich(
+      @PathVariable(value = WebEntityConstants.PATH_PARAM_TYPE) String type,
+      @PathVariable(value = WebEntityConstants.PATH_PARAM_IDENTIFIER) String identifier,
+      @RequestParam(value = "action") String action,
+      HttpServletRequest request)
```

Comment:
I added a question to specs if the profile should be added explicitly. (debug can still be implicitly used with the current implementation as it is picked up directly from the request)